inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/tenant/InlongTenantService.java

```diff
@@ -59,4 +59,11 @@ public interface InlongTenantService {
      */
     Boolean update(InlongTenantRequest request);
 
+    /**
+     * List tenant by current user
+     * @param request
+     * @return
```

Comment:
Add comment

Reply:
@vernedeng Please add description for param and return tags.

Reply:
fixed, thx